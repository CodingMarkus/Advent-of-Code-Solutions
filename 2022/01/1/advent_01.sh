#!/bin/sh

# Run as:
# cat advent_01_input.txt | sh advent_01.sh

set -e

highest=0
current=0

while read -r line
do
	if [ -z "$line" ]
	then
		[ $current -gt $highest ] && highest=$current
		current=0
	else
		current=$(( current + line ))
	fi
done

[ $current -gt $highest ] && highest=current
echo $highest