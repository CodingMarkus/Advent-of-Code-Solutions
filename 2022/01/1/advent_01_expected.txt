Answer: 24000
Answer: 68775
